modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsRelationshipsDisplayContext.java

```diff
@@ -72,6 +87,69 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems()
 				"delete", "delete", "async"));
 	}
 
+	public JSONArray getObjectRelationshipDeletionTypesJSONArray() {
+		JSONArray jsonArray = JSONFactoryUtil.createJSONArray();
+
+		return jsonArray.put(
```

Comment:
Hey Dan, 
I found a method `JSONUtil.putAll()` which returns a JSONArray, and we won't need a local variable.
I think it would be something like this:
`JSONUtil.putAll(
     JSONUtil.put(
           "default", true
      ).put(
          "label", "test"
   ),
      JSONUtil.put(
         "default", false
      ).put(
         "label", "test_2"
   ))`

or `or JSONUtil.putAll(JSONFactoryUtil.createJSONObject())` . Could you test it please?